Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -180,6 +181,8 @@ export AWS_ACCESS_KEY_ID=$(kubectl -n default get secret ceph-bucket -o jsonpath
 export AWS_SECRET_ACCESS_KEY=$(kubectl -n default get secret ceph-bucket -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 --decode)
 ```
 
+If user enabled `hosting.dnsNames` in the `CephObjectStore` CRD, then bucket can be accessed via [virtual host style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). The bucket endpoint will be `http://$BUCKET_NAME.$AWS_HOST`.
```

Comment:
We've begun trying to avoid referencing "user" (or "you") in official docs. Also, it seems worth mentioning that path-style access can (must) be used if `dnsNames` are unset.

```suggestion
If any `hosting.dnsNames` are set in the `CephObjectStore` CRD, S3 clients can access buckets in [virtual-host-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html).
Otherwise, S3 clients must be configured to use path-style access.
```